src/ucp/core/ucp_request.h

```diff
@@ -79,6 +79,19 @@ enum {
 
 
 /**
+ * Multirail rendezvous-get info.
+ * In regular rndv-get way there is always number of rkeys
+ * is equal to rndv lanes, but in case if tag offloading is
+ * available - only one rkey is proceeded
+ */
+typedef struct ucp_rndv_get_rkey {
+    ucp_lane_index_t  lane_num; /* number of rkeys obtained from peer */
```

Comment:
why need to keep "lane_num" (which should be renamed to num_lanes) and "lane_idx" in this struct, and not directly in ucp_request.send.rndv_get? this way could avoid this struct altogether:
```
            struct {
                     uint64_t             remote_address; /* address of the sender's data buffer */
                     uintptr_t            remote_request; /* pointer to the sender's send request */
                     ucp_lane_index_t     num_lanes;
                     ucp_lane_index_t     lane_idx;
                     ucp_request_t        *rreq;           /* receive request on the recv side */
                     uct_rkey_bundle_t    *rkey_bundle;      /* rendezvous-get remote keys */
            } rndv_get;
```